src/assets/stylesheets/app-palette.scss

```diff
@@ -128,7 +128,7 @@ button[data-md-color-accent] {
 
       // Repository containing source
       .md-nav__source {
-        background-color: mix($color, $md-color-black, 75%);
+        background-color: $color; // TODO: https://codyhouse.co/blog/post/how-to-combine-sass-color-functions-and-css-variables mix($color, var(--md-color-text), 75%);
```

Comment:
We should use `hsl` as a color space, because then we can easily mix colors and deduce shades.

Reply:
Yup, that's what the link is about, just not sure which parts we should implement as custom functions yet.

Reply:
I read the article and must say, that it sounds very complicated for what we're trying to achieve and it doesn't solve all of our problems, namely that users should be able to override colors with their own CSS variables. IMHO the UX should be:

1. Set one of the default Material colors via `mkdocs.yml` - this should be possible as of now
2. Override colors via CSS variables on the `:root` pseudo class or any element via extra CSS

We cannot use any SASS functions or mixins implementing this approach, because users should be able to override colors without compiling the whole theme again. This would mean that users would be obliged to define the `H`, `S` and `L` parts of the color separately, so all shades can be deduced. I guess that's too complicated and will lead to a lot of issues, so we should maybe (at the current moment) stick to `RGB`. The property definitions using the `mix` function should be normalized as separate CSS variables.